components/button/Button.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Button as ButtonComp } from "@chakra-ui/react";
+
+export interface ButtonPropsModel {
+  buttonText: string;
+  background: string;
+  color: string;
+}
+
+const Button: React.FC<ButtonPropsModel> = (props) => {
+  return (
+    <>
+      <ButtonComp
+        height={"48px"}
+        backgroundColor={props.background}
+        color={props.color}
+        width={"100%"}
+        borderRadius={"12px"}
+        border={"1px solid #A71B4A"}
```

Comment:
can we add the common color in a ts file ( util file and then use that variable everywhere ). so that if in future we have to change the color, we can change in one place

Reply:
@kundanEminds is it possible to do these things using already existing theme files of the template ? Maybe add extra variables there if it makes sense. cc. @ujjwal502

Reply:
Done